resource_management_mobile/src/app/core/constant/cookies.constants.ts

```diff
@@ -4,4 +4,8 @@ import { Injectable } from '@angular/core';
 export class CookiesConstants {
   token = 'token';
   user = 'user';
+  hiring_stage = 'hiring_stage';
```

Comment:
Add this module names in static.enum.ts.

Reply:
Fixed